include/dlaf/tune.h

```diff
@@ -53,7 +53,7 @@ struct TuneParameters {
       std::max<std::size_t>(1, pika::resource::get_thread_pool("default").get_os_thread_count() / 2);
   std::size_t red2band_barrier_busy_wait_us = 1000;
   std::size_t tridiag_rank1_nworkers =
-      std::max<std::size_t>(1, pika::resource::get_thread_pool("default").get_os_thread_count() / 2);
+      std::max<std::size_t>(1, pika::resource::get_thread_pool("default").get_os_thread_count());
```

Comment:
The `std::max` shouldn't be needed anymore, right? It was just there in case `num_threads` is `1` and `num_threads / 2` is `0`.

Reply:
See https://github.com/eth-cscs/DLA-
Future/pull/912/commits/895116962e314081234c7d8826d24067a31b3961